### PR TITLE
Create Address Form

### PR DESCRIPTION
**Pull request by LondonAtlas**
Fixed some pod strangeness. Added the Address collection view controller. Added a one time use location request. Added a map snap shooter.

address form

### CODE REVIEW

**Comment by LondonAtlas** [comment]
This view controller will be placed inside a page view controller that will contain the remaining designs.

File: StreetTreesFoundationKit/STFKMapSnapshot.swift [hunk 1]
```diff
@@ -0,0 +1,102 @@
+//
+//  STFKMapSnapshot.swift
```
Comment (WERUreo) [inline StreetTreesFoundationKit/STFKMapSnapshot.swift:2]: I'm glad you made this class!  I'll modify what I was working on to use this to create the map snapshot on the tree details view controller.

File: StreetTreesFoundationKit/UIColorExtension.swift [hunk 2]
```diff
@@ -28,7 +28,22 @@
...
+    }
+    
     public class func orlandoGreenColor(alpha: CGFloat = 1.0) -> UIColor {
-        return UIColor(red: 0.5294117647, green: 0.7333333333, blue: 0.3960784314, alpha: alpha)
+        return UIColor(red: 135.colorValue(), green: 186.colorValue(), blue: 101.colorValue(), alpha: alpha)
+    }
+    
+}
+
+extension Int {
```
Comment (WERUreo) [inline StreetTreesFoundationKit/UIColorExtension.swift:45]: This right here is why I love extensions :)

**Review by WERUreo — APPROVED** [review]
Every time I look at your code, I learn something new.  This all looks awesome.

**Review by WERUreo — APPROVED** [review]
Looks good!

**Pull request merged**